### ISSUE TITLE
Invalid passwords with Mistlands update

### ISSUE THREAD

**Issue by cafeoh** [bdew-valheim/quickconnect#8]
Quick connecting with a given password fails on the password prompt complaining it's wrong on the latest update (haven't tried on stable branch)
Copy/pasting the password directly from the configuration file to the textbox works fine. (I made sure there's no trailing spaces or anything of the sort).

**Comment by ddt3** [comment]
same issue: password is correct in cfg, server logs show: wrong password.

**Comment by riuson** [comment]
Log:
```
[Info   : Unity Log] 12/22/2022 20:31:32: Connecting to server with Steam-backend 192.168.1.99:2456

[Info   : Unity Log] 12/22/2022 20:31:32: Starting to connect to 192.168.1.99:2456

[Info   :QuickConnect] Authenticating with saved password...
[Info   : Unity Log] 12/22/2022 20:31:33: Got status changed msg k_ESteamNetworkingConnectionState_Connecting

[Info   : Unity Log] 12/22/2022 20:31:33: New connection

[Info   : Unity Log] 12/22/2022 20:31:33: Got status changed msg k_ESteamNetworkingConnectionState_Connected

[Info   : Unity Log] 12/22/2022 20:31:33: Connected

[Info   : Unity Log] 12/22/2022 20:31:33: Got connection SteamID 9____________________6

[Info   : Unity Log] 12/22/2022 20:31:33: Session auth respons callback

[Info   : Unity Log] 12/22/2022 20:31:33: Got connectoin error msg ErrorPassword

[Info   : Unity Log] 12/22/2022 20:31:33: Lost connection to server:ErrorPassword

[Info   : Unity Log] 12/22/2022 20:31:33: Shuting down
```

**Comment by billabonGS4** [comment]
password is wrong 
would be nice if you can fix it

**Comment by bdew** [comment]
Should be fixed in v1.6.0
